src/Components/CoverImage.jsx

```diff
@@ -0,0 +1,18 @@
+import HeartIcon from "../assets/icons/heart.svg";
+import PlayIcon from "../assets/icons/play.svg";
+import Dots from "../assets/icons/dots.svg";
+
+export const CoverImage = (props) => {
+  const albumImage = props.coverImages[0];
+  return (
+    <div className="cover-image-container">
+      <img className="cover-image" src={albumImage.url} alt="Music album cover image" />
+      <img src={HeartIcon} className="heart-icon" alt="Heart icon" />
+      <a href={props.href} target="_blank">
+        <img src={PlayIcon} className="play-icon" alt="Play icon" />
+      </a>
+
+      <img src={Dots} className="dots-icon" alt="Dots icon" />
+    </div>
+  );
+};
```

Comment:
Looks very clean and nice. Just a tip we learned last week was that if you use the target="_blank" you can also add the rel="noopener noreferrer" so that the opened page cannot control the original page which makes the page more safe.